### DISCUSSION TITLE
Verified Badges for organizations in The GitHub mobile app on ios

### DISCUSSION THREAD

**Discussion by doctor-ra** [community/community#2807] [Mobile]
The GitHub app does not allow you to see whether an organization has a verified domain indicated by the green verified tag next to an organizations name on the website. It would be nice for this feature to be implemented in the GitHub mobile app

**Comment by doctor-ra** [comment]
.

**Comment by candyho** [comment]
Hey thanks for sharing your feedback. I'll add this idea to our improvement backlog.

**Reply by doctor-ra** [reply]
Thanks for the reply, if you could also add the fact that the PRO tag doesn’t appear next to someone’s name when viewing their profile that would be great


Also, do u know when these features may be implemented. A general timeline is good enough for me. A year, a month, a week etc.

**Comment by doctor-ra** [comment]
Hello, I haven’t seen this feature implemented yet. Any word on its progress?

**Reply by doctor-ra** [reply]
@candyho
